docs/API Reference/Package.md

```diff
@@ -111,25 +111,6 @@ Generator that traverses all entries in the package tree and returns tuples of (
 with keys in alphabetical order.
 
 
-## Package.load(readable\_file)  {#Package.load}
```

Comment:
Why is this being deleted?

Reply:
Hm... Looks like it's because you renamed `load` to `_load`. Want to move the docstring back to `load`?

Reply:
Yeah, I'll open a PR for that